testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -472,8 +472,7 @@ public void createIndex(CqlSession session) throws IOException {
 
     ApiError response = objectMapper.readValue(body, ApiError.class);
     assertThat(response.getCode()).isEqualTo(HttpStatus.SC_BAD_REQUEST);
-    assertThat(response.getDescription())
-        .isEqualTo("Bad request: An index named test_idx already exists");
+    assertThat(response.getDescription()).contains("Index test_idx already exists");
```

Comment:
A few error messages are slightly different. I think that's acceptable as long as the status code is the same.

Reply:
This is always bit of gray area; I agree that users should not rely on failure message details. We haven't really documented those anywhere either, outside of tests.